tsconfig.json

```diff
@@ -1,14 +1,11 @@
 {
   "compilerOptions": {
-    "checkJs": true,
-    "declaration": true,
-    "emitDeclarationOnly": true,
-    "lib": ["dom", "dom.iterable", "es2020"],
-    "types": [
-      // vitest types don't work with module "node16", see
-      // https://github.com/vitejs/vite/issues/11552
-      "@types/jest"
-    ],
-    "module": "node16"
```

Comment:
You should keep `"module": "node16"`. This is the recommended (only correct really) setting for libraries built with `tsc` (which you do and should)

Reply:
One thing that I've found really helpful with testing types is https://arethetypeswrong.github.io/ (I think @remcohaszing might have actually been the one that recommended it to me!)

Reply:
I might have! It's definitely something I recommend.

But if you just build with `tsc --build` and use `"module": "node16"`, you can't really go wrong.